src/main/java/com/bieganski/ox/checkers/Judge.java

```diff
@@ -0,0 +1,45 @@
+package com.bieganski.ox.checkers;
+
+import com.bieganski.ox.App;
+import com.bieganski.ox.BoardListener;
+import com.bieganski.ox.GameListener;
+import com.bieganski.ox.model.Field;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeSet;
+
+/**
+ * Board checker, Judge initialize win checkers
```

Comment:
Czemu w dwu liniach? I czemu zaczynamy od Board checker? Jak rozumiem, Sędzia i Sprawdzacz Planszy inicjalizują Sprawdzacze Wygranej? Razem? Jak to robią? Jeden woła jedną metodę, drugi drugą? Nie mógłby jeden tego robić? Po co są dwa?

Reply:
Faktycznie nie potrzebny był Board checker. Javadoc poprawiony.